src/js/components/ServicesTable.js

```diff
@@ -147,13 +146,14 @@ var ServicesTable = React.createClass({
   },
 
   getOpenInNewWindowLink(service) {
-    if (!(service instanceof Framework) || !service.getWebURL()) {
+    if (!(service instanceof Service) || !service.getWebURL()) {
       return null;
     }
 
     return (
       <a className="table-display-on-row-hover"
-        href={Cluster.getServiceLink(service.getName())} target="_blank"
+        href={service.getWebURL()}
```

Comment:
Added a comment about why this is not possible, and why we need the check for `Service`